docs/account_access.md

```diff
@@ -1,22 +1,24 @@
-# account\_access
+# account_access
 
 [<-Back-to:Auth](database-auth.md)
 
-**The \`account\_access\` table**
```

Comment:
Actually revert this. it is to make it look like \`account_access\` instead of `account_access` since we use back ticks when using SQL code

Reply:
Updated.